src/main/java/org/folio/rest/impl/CopycatImpl.java

```diff
@@ -111,10 +112,13 @@ public void postCopycatImports(CopyCatImports entity, Map<String, String> okapiH
               log.info("Embedding identifier {} in MARC {}",
                   entity::getInternalIdentifier, () -> pattern);
               JsonMarc.embedPath(marc, pattern, entity.getInternalIdentifier());
+              name = "copycat update";
```

Comment:
final static String field instead?

Reply:
This is merged, but I bet this is going to change anyway.. See https://issues.folio.org/browse/MODCPCT-18 .. In fact it could something more dynamic, like "update with OCLC identifier xxx" or "create with OCLC identifier xxx".

Reply:
Can you not set the name? What happens then?

Reply:
A complaint.